### PR TITLE
Initial entities for pass bounded context

### PR DESCRIPTION
**Pull request by dpaquette**
Related to #17

### CODE REVIEW

File: src/AlpineSkiHouse.Web/Models/Scan.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AlpineSkiHouse.Models
+{
+    public class Scan
+    {
+        public int Id { get; set; }
+
+        public int CardId { get; set; }
```
Comment (MisterJames) [inline src/AlpineSkiHouse.Web/Models/Scan.cs:12]: Am I right in thinking that a card can have only one active pass at a time?

What if I have an annual pass for the small hill, where I'm staying, and at noon I decide to trot down the road to the other hill on a day pass?  

Are we worried about that use case? Might be too edge for the topic of our book.

Comment (dpaquette) [inline src/AlpineSkiHouse.Web/Models/Scan.cs:12]: I don't think there is anything about our model that would stop the case you described from working.

Comment (MisterJames) [inline src/AlpineSkiHouse.Web/Models/Scan.cs:12]: Hmm...well there is only a `CardId` on here. 

So to think through that...a card being scanned would not be valid given any current activations, thus the business logic would loop back to see if there was a valid, not-yet-activated pass. Okay...that seems to make sense.

**Comment by MisterJames** [comment]
:shipit:

**Pull request merged**